src/SFA.DAS.Employer.FrontDoor.Web/Views/Error/ApplicationError.cshtml

```diff
@@ -0,0 +1,18 @@
+@using SFA.DAS.Employer.FrontDoor.Web.Infrastructure
+@{ 
+    ViewData["Title"] = "Sorry, there is a problem with the service";
+    ViewData["Description"] = "There is a problem with the service. Try again later.";
+    @*ViewData[nameof(BreadCrumbViewModel)] = new BreadCrumbViewModel {Level = BreadCrumbLevel.Error};*@
+}
+
+<main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+            <p class="govuk-body">Try again later.</p>
+            <p class="govuk-body"><a href="https://help.apprenticeships.education.gov.uk/hc/en-gb#contact-us" class="govuk-link govuk-link--no-visited-state">Contact the Apprenticeship Service</a> if you need help.</p>
```

Comment:
Not sure if the apprenticeship service is where the user should get help. Question for Becky. (Same for the 404 page.)

Reply:
Will add a story for this.